peachjam/settings.py

```diff
@@ -65,7 +65,7 @@
     "sass_processor",
     "import_export",
     "treebeard",
-    "background_task",
+    # "background_task",
```

Comment:
I think this needs to be added back?

Reply:
Yes, I've added it back. It was part of the deploy troubleshooting.